assets/js/modules/analytics-4/datastore/__fixtures__/available-audiences.json

```diff
@@ -0,0 +1,38 @@
+[
```

Comment:
This file is copied in from the [PR](https://github.com/google/site-kit-wp/pull/8606/files#diff-14a74e8381f52f6f42e160be1dff0079e1d0e8bbc7b222bb035f49a1f42621ab) for #8487, I haven't branched from that branch as there are pending changes to apply to the file there.